zap.conf

```diff
@@ -121,3 +121,4 @@
 90034	WARN	(Cloud Metadata Potentially Exposed - Active/beta)
 10202	OUTOFSCOPE	http://.*:3000/events.*
 20012	OUTOFSCOPE	http://.*:3000/events.*
+10062	OUTOFSCOPE	http://.*:3000/users/auth/github
```

Comment:
This change removes the check for PII Disclosure on the github login page, and it shows the limits of automated scanning. We were having some failures on this rule (PII disclosure) and route where the scanner thought that it found a Diners Club International credit card number. It's (potentially) true that it is a valid Diners Club card number, but that's not what it was in context.